src/main/java/org/folio/config/ServicesConfiguration.java

```diff
@@ -208,14 +209,20 @@ public LedgerRolloverService ledgerRolloverService(FiscalYearService fiscalYearS
     RolloverProgressService rolloverProgressService,
     RolloverErrorService rolloverErrorService,
     PostgresFunctionExecutionService postgresFunctionExecutionService,
+    UniqueValidationService uniqueValidationService,
```

Comment:
`UniqueValidationService` seems to imply that it can uniquely validate anything. If this is not the case please change the name to show that it does unique validation on something specific.